packages/audio-block/src/components/BlockAttachments.tsx

```diff
@@ -1,12 +1,11 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { Attachments, useAttachments } from '@frontify/guideline-blocks-settings';
+import { Attachments, useAttachmentsContext } from '@frontify/guideline-blocks-settings';
 import { BlockAttachmentsProps } from '../types';
-import { ATTACHMENTS_ASSET_ID } from '../settings';
 
 export const BlockAttachments = ({ appBridge }: BlockAttachmentsProps) => {
     const { attachments, onAddAttachments, onAttachmentDelete, onAttachmentReplace, onAttachmentsSorted } =
```

Comment:
```suggestion
    const { attachments, onAttachmentsAdd, onAttachmentDelete, onAttachmentReplace, onAttachmentsSorted } =
```
So it fits the rest of the properties?

Reply:
fixed in the brandsdk